### ISSUE TITLE
Broken npm package: rollup-plugin-lucy.mjs seems to be missing

### ISSUE THREAD

**Issue by mattmoos** [matthewp/rollup-plugin-lucy#1]
Hi, just saw your post about lucy on twitter and immediately liked it. I started to try it out in a vite.js project but didn't get it to work by installing it via
```
npm install @lucy/rollup-plugin --save-dev
```

I noticed that the installed package in `node_modules/@lucy/rollup-plugin` only contains a `package.json` as well as `readme.md`. The actual plugin seems to be missing.

**Comment by matthewp** [comment]
Oops! Fixing.

**Comment by matthewp** [comment]
@mattmoos ok please true again, just released 0.1.2

**Comment by mattmoos** [comment]
@matthewp Thanks for the fast response. `rollup-plugin-lucy.mjs` is now present. 
However, it cannot be resolved. The `main` entry in the plugins `package.json` is `"main": "rollup-plugin-lucy.js"`. Should it end on `.mjs` instead?

**Comment by matthewp** [comment]
@mattmoos Ack! this is what I get for only testing the plugin locally so far! I'll update that.

**Comment by matthewp** [comment]
@mattmoos Ok, try again with 0.1.3. I should note that I think it only works with a `.mjs` vite config at the moment. This is because Vite tries to compile any other file extension and its not able to bundle Lucy, I think because Lucy uses wasm.  See this example: https://github.com/matthewp/rollup-plugin-lucy/blob/main/demos/vite-project/vite.config.mjs

**Comment by mattmoos** [comment]
@matthewp Still not working, VSCode now prints the warning
```
String does not match the pattern of "^\./".
```
for the `exports` entry in the `package.json`. When I change it to 
```
  "exports": "./rollup-plugin-lucy.mjs",
```
I get it to work. I'm using 'vite.config.mjs' as you have also written here: <https://lucylang.org/install/>

**Comment by matthewp** [comment]
@mattmoos Thanks for putting up with me. Just published `0.1.4` with that change.

**Comment by mattmoos** [comment]
@matthewp Thank you very much for your quick support, now it seems to be working as expected 👍
